README.md

```diff
@@ -1,27 +1,13 @@
-# Arrow Contracts
-
-Implementation of the Arrow token and governance smart contracts.
+# Arrow Contract
```

Comment:
Why are we changing the README to remove information?

Reply:
This README was auto-generated by hardhat, I will change it back to the original version with added instructions on running tests in hardhat.